src/routes/commercialRegistrations/types/registration.ts

```diff
@@ -0,0 +1,7 @@
+export type Registration = {
+  registration_number: string;
```

Comment:
and name

Reply:
True: https://github.com/kontist/mock-solaris/pull/197/commits/75201bba2c803fadf50223c9172ec69f1f86a764